Terminal.Gui/ConsoleDrivers/CursesDriver/UnmanagedLibrary.cs

```diff
@@ -264,20 +261,20 @@ private static string FirstValidLibraryPath (string [] libraryPathAlternatives)
 
     private static class Windows
     {
-        [DllImport ("kernel32.dll")] internal extern static nint GetProcAddress (nint hModule, string procName);
-        [DllImport ("kernel32.dll")] internal extern static nint LoadLibrary (string filename);
+        [DllImport ("kernel32.dll")] internal static extern nint GetProcAddress (nint hModule, string procName);
```

Comment:
More attributes on same line as method signatures and extensions in library names.

Reply:
I'm going to stop commenting on that particular problem, since one settings tweak will fix them all at once, so it's not really necessary for me to call them all out.